app/controllers/admin/creator_settings_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Admin
+  class CreatorSettingsController < Admin::ApplicationController
+    before_action :extra_authorization, only: [:create]
+
+    ALLOWED_PARAMS = %i[community_name logo_svg primary_brand_color_hex invite_only_mode public].freeze
+
+    def new; end
+
+    def create
+      current_user.saw_onboarding = true
+      current_user.save!
```

Comment:
We should be able to combine these two lines:

```suggestion
      current_user.update!(saw_onboarding: true)
```